operator/v1/types_network.go

```diff
@@ -433,6 +447,19 @@ type OVNKubernetesConfig struct {
 	// fields within ipv4 for details of default values.
 	// +optional
 	IPv6 *IPv6OVNKubernetesConfig `json:"ipv6,omitempty"`
+
+	// routeAdvertisements determines if the functionality to advertise cluster
+	// network routes through a dynamic routing protocol, such as BGP, is
+	// enabled or not. This functionality is configured through the
+	// ovn-kubernetes RouteAdvertisements CRD. Requires the 'FRRK8s' routing
+	// capability provider to be enabled as an additional routing capability.
+	// Allowed values are "Enabled", "Disabled" and ommited. When omitted, this
+	// means the user has no opinion and the platform is left to choose
+	// reasonable defaults. These defaults are subject to change over time. The
+	// current default is "Disabled".
+	// +openshift:enable:FeatureGate=RouteAdvertisements
+	// +optional
+	RouteAdvertisements RouteAdvertisementsEnablement `json:"routeAdvertisements,omitempty"`
```

Comment:
Is it possible to disable this once it has been enabled, what would happen?

Reply:
Yes. Pretty much the same thing as when it is enabled: OVN-K will be rolled out with the corresponding configuration, which will in itself cause a reconfiguration of the cluster network as it was before it being enabled.

Reply:
Ok, so, there should be no serious concern about a user disabling this?

Reply:
Nothing permanent in regards to the operation of the cluster itself. 

Network reconfiguration, both when enabling and disabling, might cause short temporary network hiccups depending on what kind of reconfiguration needs to be done, which depends on the instances of the CRD in the system when it happens (just as it would happen when the user creates/deletes instances of the CRD itself when this is enabled)